zproject/email_backends.py

```diff
@@ -108,3 +109,9 @@ def send_messages(self, email_messages: Sequence[EmailMessage]) -> int:
                 email_log_url = settings.ROOT_DOMAIN_URI + "/emails"
                 logging.info("Emails sent in development are available at %s", email_log_url)
         return num_sent
+
+
+class CustomSMTPEmailBackend(EmailBackend):
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)  # nocoverage
+        self.opened_at = timezone_now()  # nocoverage
```

Comment:
Should it be okay to skip coverage in cases where the class is being mocked by MagicMock?

Reply:
I think you can use the `time_machine` library to mock time without mocking this function, and that'll remove the ened to use `nocoverage`.

Reply:
Since the mocked object was needed for the original test, I've added the following extra lines to the test file to add coverage:


```js
        # Test opened_at is being set to timezone_now
        with time_machine.travel(
            timezone_now(),
            tick=False,
        ):
            custom_backend = CustomSMTPEmailBackend()
            self.assertEqual(custom_backend.opened_at, timezone_now())
 ```